test/Transfer/Adapter/AbstractTest.php

```diff
@@ -301,7 +301,7 @@ public function testGetFilterShouldReturnNullWhenNoMatchingIdentifierExists()
 
     public function testAdapterShouldAllowPullingFiltersByFile()
     {
-        $this->adapter->addFilter('Boolean', 1, 'foo');
+        $this->adapter->addFilter('Boolean', [1], 'foo');
```

Comment:
On investigation, this is due to how "creation options" are handled in v2 with the invokable factory. I think there may be a way to work around it; testing something now.